### ISSUE TITLE
mkdocs-material's tags plugin doesn't fully work with mkdocs-static-i18n

### ISSUE THREAD

**Issue by markszabo** [ultrabug/mkdocs-static-i18n#177]
[mkdocs-material](https://github.com/squidfunk/mkdocs-material) has a [built-in tags plugin](https://squidfunk.github.io/mkdocs-material/setup/setting-up-tags/), however it doesn't fully work with mkdocs-static-i18n.

# Minimal example

https://github.com/markszabo/mkdocs_i18n_tags_bug 

# Things that work

### Tags show up on the top of the pages in all languages:

![image](https://user-images.githubusercontent.com/12611720/212291911-0ac6cac3-d1d8-421e-9269-9410708915a1.png)

![image](https://user-images.githubusercontent.com/12611720/212291948-f8858dae-41de-4327-8a2e-0cd1c1aa26f2.png)

### In the main language, the tags page works and shows the tags from the same language

![image](https://user-images.githubusercontent.com/12611720/212292032-c3220afb-d43c-4838-9cba-45191fadec81.png)

# Thing that doesn't work

### In other languages the tags page is broken

![image](https://user-images.githubusercontent.com/12611720/212292315-297e71c1-30a6-4e0e-8115-5b3fb22f3d4d.png)

### The tag links on top of the pages always point to the tags page of the main language

![image](https://user-images.githubusercontent.com/12611720/212292474-053d9a6d-6a9a-4952-8483-42d8ca1e347f.png)

# Steps to reproduce

1. Clone the https://github.com/markszabo/mkdocs_i18n_tags_bug repo
2. Run the mkdocs server, e.g. `./dev-server.sh`
3. Check that http://127.0.0.1:8000/ja/tags/ doesn't show the tags

**Comment by ultrabug** [comment]
Thanks for this nice and clear report, I'll check that out

**Comment by champsblanc** [comment]
I have the same problem but i can't even manage to render the tags index in either language. Tags at the top of md documents seem to work fine.

**Comment by ultrabug** [comment]
Just checking in to acknowledge and apologize for the fact that I did not properly look into this yet.

Once #187 is done, I'll hope to check that out.

**Comment by usulpt** [comment]
+1 here, couldn't even render the tags in the primary language, found a workaround by excluding the second language files from input (using https://github.com/apenwarr/mkdocs-exclude).

**Comment by ultrabug** [comment]
Hello I'd like to point out that the upcoming 1.0.0 rework of the plugin happening in #216 will fix this issue. I just tested it and it works properly!

If you're interested, please give a try to #216 and report your experience in the PR!

**Comment by ultrabug** [comment]
I'm happy to announce release v1.0.0 of the `mkdocs-static-i18n` plugin!

Please check it out as it should fix this issue! Reopen if needed.

**Comment by markszabo** [comment]
Thank you very much, and sorry for the delay. I finally got to test it today, and run into an issue. I'll open a new issue about it.
